src/Microsoft.VisualStudio.Editors/PropPages/CSharpLanguageVersion.vb

```diff
@@ -8,6 +8,9 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
     Friend Class CSharpLanguageVersion
 
         Private Const s_languageVersion_Default As String = "default"
+        Private Const s_languageVersion__DisplayNameForDefault As String = "C# latest major version (default)"
```

Comment:
@davkean I'm having troubles loading up the VB forms designer so that I can hook up the generated strings to the form correctly. Will update this PR as soon as I figure it out.